Makefile

```diff
@@ -1,49 +1,25 @@
-app_name=bruteforcesettings
```

Comment:
ah I was to agressive in replacing it seems... lets see if I can somehow merge this all :)